include/swift/AST/DiagnosticsSema.def

```diff
@@ -140,8 +140,8 @@ ERROR(could_not_use_instance_member_on_type,none,
       "%select{| instance of nested}3 type %0",
       (Type, DeclNameRef, Type, bool))
 ERROR(could_not_use_member_on_existential,none,
-      "member %1 cannot be used on value of protocol type %0; use a generic"
-      " constraint instead",
+      "member %1 cannot be used on value of protocol type %0; consider using a"
```

Comment:
Nit: drop `a` per diagnostics guide.

Reply:
Mmm, I think ‘a’ here actually can’t be dropped; the rules of reduced written register in English are a bit hard to codify but this particular article isn’t droppable.

Reply:
Let's leave it in place then :)